bindsnet/pipeline/__init__.py

```diff
@@ -183,9 +206,26 @@ def plot_obs(self) -> None:
             self.obs_ax.set_title('Observation')
             self.obs_ax.set_xticks(())
             self.obs_ax.set_yticks(())
-            self.obs_im = self.obs_ax.imshow(self.obs, cmap='gray')
+            self.obs_im = self.obs_ax.imshow(self.env.reshape(), cmap='gray')
```

Comment:
What does this do?

Reply:
When you want to get MNIST data from environment as self.obs, its shape is flattened. So imshow() cannot take that data. And there was env.reshape() which returns reshaped (for example, [28,28]) array so I replaced self.obs with env.reshape()